src/front/table/Table.jsx

```diff
@@ -24,7 +24,7 @@ module.exports = React.createClass({
 
   getInitialState() {
     return {
-      perPage: this.props.perPage,
+      perPage: parseInt(window.localStorage.getItem('perPage') || this.props.perPage),
```

Comment:
This means that the setting will be kept for *every* table in the app. Is that required behavior?